templates/components/set-password-migration-form.jsx

```diff
@@ -0,0 +1,42 @@
+var React = require('react');
+var validators = require('../lib/validatorset');
+var Form = require('./form/form.jsx');
+var IconText = require('./icontext.jsx');
+
+var fields = [
+  {
+    'password': {
+      'placeholder': 'Enter your new password',
+      'type': 'text',
+      'validator': 'username',
+      'errorMessage': 'Invalid username'
+    }
+  }
+];
+
+var fieldsValidators = validators.getValidatorSet(fields);
+
+var SetPasswordMigration = React.createClass({
+  render: function() {
+    return (
+      <div className="migrateKeyContainer">
+        <IconText
+          iconClass="emailSentIcon fa fa-lock"
+          className="emailSent arrow_box">
+            <p className="emailSentHeader">Set your password</p>
```

Comment:
the IconText element has a special attribute for headers, so do we want a `<p>` here, or should this be the proper header?